code/game/objects/items/trash.dm

```diff
@@ -8,84 +8,86 @@
 	desc = "This is rubbish."
 
 /obj/item/trash/raisins
-		name = "4no raisins"
-		icon_state= "4no_raisins"
+	name = "4no raisins"
+	icon_state= "4no_raisins"
 
 /obj/item/trash/candy
-		name = "Candy"
-		icon_state= "candy"
+	name = "Candy"
+	icon_state= "candy"
 
 /obj/item/trash/cheesie
-		name = "Cheesie honkers"
-		icon_state = "cheesie_honkers"
+	name = "Cheesie honkers"
+	icon_state = "cheesie_honkers"
 
 /obj/item/trash/chips
-		name = "Chips"
-		icon_state = "chips"
+	name = "Chips"
+	icon_state = "chips"
 
 /obj/item/trash/popcorn
-		name = "Popcorn"
-		icon_state = "popcorn"
+	name = "Popcorn"
+	icon_state = "popcorn"
 
 /obj/item/trash/sosjerky
-		name = "Scaredy's Private Reserve Beef Jerky"
-		icon_state = "sosjerky"
+	name = "Scaredy's Private Reserve Beef Jerky"
+	icon_state = "sosjerky"
 
 /obj/item/trash/syndi_cakes
-		name = "Syndi cakes"
-		icon_state = "syndi_cakes"
+	name = "Syndi cakes"
+	icon_state = "syndi_cakes"
 
 /obj/item/trash/waffles
-		name = "Waffles"
-		icon_state = "waffles"
+	name = "Waffles"
+	icon_state = "waffles"
 
 /obj/item/trash/plate
-		name = "Plate"
-		icon_state = "plate"
+	name = "Plate"
+	icon_state = "plate"
 
 /obj/item/trash/snack_bowl
-		name = "Snack bowl"
-		icon_state	= "snack_bowl"
+	name = "Snack bowl"
+	icon_state	= "snack_bowl"
 
 /obj/item/trash/pistachios
-		name = "Pistachios pack"
-		icon_state = "pistachios_pack"
+	name = "Pistachios pack"
+	icon_state = "pistachios_pack"
 
 /obj/item/trash/semki
-		name = "Semki pack"
-		icon_state = "semki_pack"
+	name = "Semki pack"
+	icon_state = "semki_pack"
 
 /obj/item/trash/tray
-		name = "Tray"
-		icon_state = "tray"
+	name = "Tray"
+	icon_state = "tray"
 
 /obj/item/trash/candle
-		name = "candle"
-		icon = 'icons/obj/candle.dmi'
-		icon_state = "candle4"
+	name = "candle"
+	icon = 'icons/obj/candle.dmi'
+	icon_state = "candle4"
 
 /obj/item/trash/liquidfood
-		name = "\improper \"LiquidFood\" ration"
-		icon_state = "liquidfood"
+	name = "\improper \"LiquidFood\" ration"
+	icon_state = "liquidfood"
 
 /obj/item/trash/can
-		name = "crushed can"
-		icon_state = "cola"
+	name = "crushed can"
+	icon_state = "cola"
+	var/is_glass = 0
+	var/is_plastic = 0
 
 /obj/item/trash/gum
-		name = "chewed gum"
-		desc = "NOT free candy."
-		icon_state = "gum"
+	name = "chewed gum"
+	desc = "NOT free candy."
+	icon_state = "gum"
 
 /obj/item/trash/tastybread
-		name = "bread tube"
-		icon_state = "tastybread"
+	name = "bread tube"
+	icon_state = "tastybread"
 
 /obj/item/trash/tapetrash
-		name = "old duct tape"
-		icon_state = "tape"
-		desc = "Not sticky anymore."
-		throw_range = 1

```

Comment:
Someone tried to make the paths not relative, but didn't adjust the indentation for the vars on the subtypes. Byond was insane enough to allow it anyways.